test/promql/tests/test_notready_pods.yaml

```diff
@@ -0,0 +1,88 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+
+      # Pod is Pending, not unschedulable and its alerted
+      - series: 'kube_pod_status_phase{pod="pod-1", namespace="ns-1", phase="Pending"}'
+        values: '0x15'
+      - series: 'kube_pod_status_unschedulable{namespace="ns-1", pod="pod-1"}'
+        values: '0x15'
+
+      # Pod is Pending, unschedulable and its alerted
```

Comment:
This is not alerted. Please change the comment to not alerted.

Reply:
> This is not alerted. Please change the comment to not alerted.

Updated